analysis/study_definition.py

```diff
@@ -54,27 +68,6 @@
             "category": {"ratios": {"M": 0.49, "F": 0.51}},
         }
     ),
-    
-    # https://codelists.opensafely.org/codelist/opensafely/chronic-cardiac-disease/2020-04-08/
-    chronic_cardiac_disease=patients.with_these_clinical_events(
-        chronic_cardiac_disease_codes,
-        returning="date",
-        find_first_match_in_period=True,
-        include_month=True,
-        return_expectations={"incidence": 0.2},
-    ),
-
-    # https://codelists.opensafely.org/codelist/opensafely/chronic-liver-disease/2020-06-02/
-    chronic_liver_disease=patients.with_these_clinical_events(
-        chronic_liver_disease_codes,
-        returning="date",
-        find_first_match_in_period=True,
-        include_month=True,
-        return_expectations={
-            "incidence": 0.2,
-            "date": {"earliest": "1950-01-01", "latest": "today"},
-        },
-    ),
 
     # https://github.com/opensafely/risk-factors-research/issues/51
     bmi=patients.most_recent_bmi(
```

Comment:
on BMI I have built categories into study def here in case useful https://github.com/opensafely/covid-vaccine-preliminary-study/blob/6a3391681f2613aea30609d546a2673b80a14977/analysis/study_definition_delivery.py#L300

Reply:
I have changed - thank you